Day4/App.js

```diff
@@ -4,9 +4,16 @@ import { title } from './constants.js';
 import SearchBar from './SearchBar.js';
 import { useEffect, useState } from 'react';
 import NoResultsComponent from './NoResultsComponent.js';
-import { createBrowserRouter, RouterProvider, Outlet, Link } from 'react-router-dom';
+import {
```

Comment:
Just for readability, try to push all your global import to the top. Kind of in a sequence like -> global imports, module imports, css imports and so on

Reply:
Resolved